tools/add_to.hpp

```diff
@@ -142,50 +142,50 @@ namespace antler {
       add_to_project(CLI::App& app) {
          path = system::fs::current_path().string();
          subcommand = app.add_subcommand("add", "Add an app, dependency, library or test to your project.");
-         subcommand->add_option("-p, path", path, "Path containing the project's yaml file.");
+         subcommand->add_option("-p", path, "Path containing the project's yaml file.")->default_val(".");
 
          app_subcommand = subcommand->add_subcommand("app", "Add a new app to your project.");
          app_subcommand->footer(std::string(R"(Examples:)")
                + "\n\t" + app.get_name() +R"( add app MyApp C++ \\-O2)"
                + "\n\t" + app.get_name() +R"( add app -p ./path-to-project/ -n MyApp -l C++ --comp -O2)"
                + "\n\t" + app.get_name() +R"( add app -n MyApp -l C++ "\-O2 -WError" \\-s)");
 
-         app_subcommand->add_option("-p", path, "Path containing the project's yaml file.");
-         app_subcommand->add_option("-n, name", obj_name, "The name of the app to add.")->required();
-         app_subcommand->add_option("-l, lang", lang, "Language this app will use.")->required();
-         app_subcommand->add_option("--comp, compile_options", copts, "Options for the compiler for this app.")
+         app_subcommand->add_option("-p", path, "Path containing the project's yaml file.")->default_val(".");

```

Comment:
I think adding the default value is okay. But if we do this, we probably need to remove line 143 above:
  `path = system::fs::current_path().string();`
  
I'm okay we either approach, let's just be consistent across the tool files.